addons/account/models/account.py

```diff
@@ -1190,7 +1190,7 @@ class AccountTaxGroup(models.Model):
 class AccountTax(models.Model):
     _name = 'account.tax'
     _description = 'Tax'
-    _order = 'sequence,id'
+    _order = 'active desc, sequence,id'
```

Comment:
@hpr-odoo Was this the case before?

Reply:
@jco-odoo , No it wasn't case before, but it was asked to show active taxes above inactive in spec.

Reply:
@hpr-odoo You will have to remove it anyways, because you can not do that in stable